exporter/dynatraceexporter/internal/serialization/histogram_test.go

```diff
@@ -131,4 +131,149 @@ func Test_serializeHistogram(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Equal(t, "prefix.min_max_hist gauge,min=3,max=7,sum=10,count=2 1626438600000", got)
 	})
+
```

Comment:
I didn't find tests covering the `sum` estimation. I guess we need to add them right?

Reply:
I'm working on those today

Reply:
6b377d14a3b3f1a480b1c7840cd22d7d8b6d207e